protected/humhub/libs/SelfTest.php

```diff
@@ -160,9 +160,10 @@ public static function getResults()
         }
 
         // Checks PCNTL Extension
-        $title = 'PHP - PCNTL Extension';
-
-        if (function_exists('pcntl_signal')) {
+	$title = 'PHP - PCNTL Extension';
+	// PCNTL may be disabled in webserver context but available in php-cli
+	// where it is actually needed.
+        if (function_exists('pcntl_signal') || shell_exec('php -r \'echo function_exists("pcntl_signal");\'') == '1') {
```

Comment:
To be honest `shell_exec()` can't be used by all systems in some shared hosts so I'd recommend first doing an `ini_get()` to check if permissions allow for it to be used.

Reply:
Yeah, I was wondering whether that would be a problem, not sure how to reproduce and test this though: try/catch / ini_get / function_exists('shell_exec')? Maybe a small function with try/catch could be the most solid solution, what do you think?

Reply:
It would be more logical to generate a `try` and then `catch` for unexpected results that could happen

Reply:
As to check `ini_get('disable_functions')` should do to display disabled functions but not sure if this is too specific or not 🤔

Reply:
Checking whether 'shell_exec' is available is apparently a bit more complicated.
https://stackoverflow.com/questions/21581560/php-how-to-know-if-server-allows-shell-exec

Important would be that no fatal error is triggered if the function is not available.